core/build.gradle

```diff
@@ -6,7 +6,7 @@ repositories {
 }
 
 dependencies {
-    compile 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.1'
+    compile 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.0'
```

Comment:
and also I'd say adding the demo project should be a separate PR as well to reduce the noise (fix vs. adding a demo project)